### PR TITLE
docs(newrelic): update the help screens for consistency

### PR DESCRIPTION
**Pull request by ctrombley**
This PR updates the help screens for consistency.

### CODE REVIEW

File: internal/config/command.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ var (
 // Command is the base command for managing profiles
 var Command = &cobra.Command{
 	Use:   "config",
-	Short: "Subcommands to manage the configuration of this tool",
+	Short: "Manage the configuration of this tool",
```
Comment (sanderblue) [inline internal/config/command.go]: Not a blocker...

```suggestion
	Short: "Manage the configuration of the New Relic CLI",
```

File: internal/credentials/command.go [hunk 2]
```diff
@@ -17,16 +17,15 @@ var (
 // Command is the base command for managing profiles
 var Command = &cobra.Command{
 	Use:   "profiles",
-	Short: "Subcommands to mange the credential profiles for this tool",
+	Short: "Manage the credential profiles for this tool",
```
Comment (sanderblue) [inline internal/credentials/command.go]: Not a blocker, similar to other suggestion...
```suggestion
	Short: "Manage the credential profiles for the New Relic CLI",
```

Comment (ctrombley) [reply internal/credentials/command.go]: 👍

File: internal/config/command.go (hunk 1 above)
Comment (ctrombley) [reply internal/config/command.go]: 👍

**Pull request merged**